enhancements/agent-installer/simplified-registry-less-cluster-operations.md

```diff
@@ -0,0 +1,426 @@
+control planes nodes. In particular, it will be used for:
+
+* Triggering a new release image copy task - as a preliminary upgrade step
+* Delete from disk a release image version not anymore in use - as an optional
+  post-upgrade step
+* Enabled/disable the registry services
+
+This is an example of how the CR will look like:
+
+```
+apiVersion: releases.openshift.io/v1alpha1
+kind: InternalReleaseImage
+metadata:
+  name: cluster
+spec:
+  releases:
+    - "quay.io/openshift-release-dev/ocp-release:4.18.0-x86_64"
+    - "quay.io/openshift-release-dev/ocp-release:4.19.0-x86_64"
+  enabled: true
+status:
+  releases:
+    - "quay.io/openshift-release-dev/ocp-release:4.18.0-x86_64"
+    - "quay.io/openshift-release-dev/ocp-release:4.19.0-x86_64"
+  enabled: true
+  conditions:
+    - type: "Available"
+      status: "True"
+      lastTransitionTime: "2025-07-12T14:02:00Z"
+      reason: "ReleasesSynchronized"
+      message: "The previously managed releases are available."
+    - type: "Progressing"
+      status: "True"
+      lastTransitionTime: "2025-07-12T13:13:00Z"
+      reason: "AddingRelease"
+      message: "Adding new release image quay.io/openshift-release-dev/ocp-release:4.19.0-x86_64 on master-0."
+  ...
+```
+
+* The `spec.releases` will be used to configure the desired release contents.
+  Adding (or removing) an entry from the list will trigger an update on all the
+  control plane nodes.
+* The `spec.enabled` will be used to turn on / off the feature. In particular,
+  when turned off, all the registry services running on the control plane nodes
+  will be stopped.
+* The `status.releases` field will report the currently managed releases
+* The `status.conditions` field will be used to keep track of the activity
+  performed on the various nodes.
+
+### Topology Considerations
+
+Will function identically in all topologies.
+
+#### Hypershift / Hosted Control Planes
+
+N/A
+
+#### Standalone Clusters
+
+N/A
+
+#### Single-node Deployments or MicroShift
+
+N/A
+
+### Implementation Details/Notes/Constraints
+
+#### Extended RHCOS ISO build
+
+The release image and operators images will be stored inside the extended RHCOS
+ISO in the storage format used by the [distribution/distribution](https://github.com/distribution/distribution)
+registry’s filesystem storage plugin.
+
+#### Distribution/distribution registry OCP integration
+
+The [distribution/distribution](https://github.com/distribution/distribution)
+registry will be used to run the registry services on each control plane
+node. The binary will be built and included in the [openshift/image-registry](https://github.com/openshift/image-registry)
+repo image, so that it will be available within an OCP release image payload.
+
+#### Installation - bootstrap phase
+
+![alt bootstrap phase](simplified-operations-bootstrap.png)
+
+Initially the registry image will be placed into a read-only [additional image
```

Comment:
Currently we simply use 'podman load' on the registry tar file.
Are you referring to the same flow?